analyzers/src/SonarAnalyzer.VisualBasic/Rules/Hotspots/DoNotHardcodeCredentials.cs

```diff
@@ -69,7 +69,7 @@ private sealed class VariableDeclarationBannedWordsFinder : CredentialWordsFinde
             public VariableDeclarationBannedWordsFinder(DoNotHardcodeCredentialsBase<SyntaxKind> analyzer) : base(analyzer) { }
 
             protected override string GetAssignedValue(VariableDeclaratorSyntax syntaxNode, SemanticModel semanticModel) =>
-                syntaxNode.Initializer?.Value.GetStringValue(semanticModel);
+                syntaxNode.Initializer?.Value.StringValue(semanticModel);
```

Comment:
Let's fix the coverage here and on C# too by removing the `?`. This is called after `ShouldHandle` and `ShouldHandle` would never return `true` in case the initializer is null. So we're sure the condition is unreachable and we can remove it.